src/Linear/Linear.tsx

```diff
@@ -141,6 +155,12 @@ export default class Linear extends React.Component<LinearProps> {
       if (zoomed) {
         blockHeight += showComplement ? lineHeight : 0; // double for complement + 2px margin
       }
+      if (primerFwdRows[i].length) {
+        blockHeight += primerFwdRows[i].length * lineHeight;
```

Comment:
because it's a row of rows, we have to multiple the row count by lineHeight (in case there's >1 rows of primers)

Reply:
Sorry, I didn't get it. Should I change something?

Reply:
No this and the other comments are just me sharing the things I changed as an FYI about some internal details